scripts/sweep_common.py

```
@@ -0,0 +1,746 @@
+
+
+# ---------------------------------------------------------------------------
+# Running a sweep
+# ---------------------------------------------------------------------------
+
+
+def subprocess_run(cmd: List[str]) -> str:
+    """Run a command, returning combined stdout+stderr (logging may go to either)."""
+    import subprocess
+
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)